local-volume/provisioner/deployment/docker/Dockerfile

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM centos:7
+FROM k8s.gcr.io/debian-base-amd64:0.4.0
 
-RUN yum update -y && \
-    yum install -y e2fsprogs && \
-    yum install -y hdparm && \
-    yum clean all
+RUN clean-install \
+    util-linux \
+    e2fsprogs \
+    bash
```

Comment:
hmm would prefer to avoid adding bash to the actual shipped image if possible.  Could we make a special image from the prod image just for testing and inject bash there?

Reply:
Oh, I see, these scripts need to be executed in the container too when you're not using jobs for cleaning.  Then this makes sense. I thought it was only for testing.